Makefile

```diff
@@ -91,6 +91,17 @@ edgeimage:
 	--build-arg RUN_FROM=${ARCH}/docker:dind \
 	-f build/edge/Dockerfile .
 
+.PHONY: edgesiteimage
+edgesiteimage:
+	mkdir -p ./build/edgesite/tmp
+	rm -rf ./build/edgesite/tmp/*
```

Comment:
first we create the tmp and then delete nothing inside it ?

Reply:
This is to handle cases in which the image has already been built before i.e. the tmp folder has contents inside. Something similar to this has been done while building edge_core image as well  https://github.com/kubeedge/kubeedge/blob/4d44da1ce4d9c001a814beaa44d0b32e257554db/Makefile#L86

Reply:
then why are creating the directory ?

Reply:
the tmp folder does not exist by default, we create it the first we build the image and used for 
 downloading and using QEMU